### PR TITLE
added usb head cam to bringup

### PR DESCRIPTION
**Pull request by benmaidel**

### CODE REVIEW

File: cob_bringup/drivers/usb_camera_node.launch [hunk 1]
```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0"?>
+
+<launch>
+  <arg name="camera_name" default="head_cam"/>
```
Comment (floweisshardt) [inline cob_bringup/drivers/usb_camera_node.launch]: let's set the default to usb_cam

File: cob_bringup/drivers/usb_camera_node.launch [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+  
+  <node name="$(arg camera_name)" pkg="usb_cam" type="usb_cam_node" output="screen" >
+    <param name="video_device" value="/dev/video0" />
+    <param name="image_width" value="2048" />
+    <param name="image_height" value="1536" />
+
+    <param name="pixel_format" value="mjpeg" />
+    <param name="camera_frame_id" value="usb_cam" />
+    <param name="framerate" value="20"/>
+    <param name="camera_name" value="head_camera"/>
```
Comment (floweisshardt) [inline cob_bringup/drivers/usb_camera_node.launch]: this should use the launch arg `"$(arg camera_name)"`

File: cob_bringup/drivers/usb_camera_node.launch [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+<launch>
+  <arg name="camera_name" default="head_cam"/>
+  
+  <node name="$(arg camera_name)" pkg="usb_cam" type="usb_cam_node" output="screen" >
+    <param name="video_device" value="/dev/video0" />
+    <param name="image_width" value="2048" />
+    <param name="image_height" value="1536" />
+
+    <param name="pixel_format" value="mjpeg" />
+    <param name="camera_frame_id" value="usb_cam" />
```
Comment (floweisshardt) [inline cob_bringup/drivers/usb_camera_node.launch]: we should use `"$(arg camera_name)_link"`, same as in https://github.com/ipa320/cob_robots/blob/indigo_dev/cob_bringup/drivers/openni2.launch

**Comment by floweisshardt** [comment]
is the head_cam_link integrated into the head urdf? @ipa-nhg?

**Comment by benmaidel** [comment]
@ipa-fmw @ipa-nhg 
nope the link is not integrated into the head urdf

File: cob_bringup/robots/cob4-2.xml [hunk 4]
```diff
@@ -266,6 +266,14 @@
 			<arg name="robot" value="$(arg robot)" />
 		</include>
 
+		<include file="$(find cob_bringup)/drivers/usb_camera_node.launch" />
+
+                <include file="$(find cob_bringup)/drivers/image_flip.launch">
+                        <arg name="robot" value="$(arg robot)"/>
+                        <arg name="pkg_hardware_config" value="$(arg pkg_hardware_config)"/>
```
Comment (floweisshardt) [inline cob_bringup/robots/cob4-2.xml]: the launch argument pkg_hardware_config is not set within cob4-2.xml. It has a default value within image_flip.launch which is good for all default cob setups.

So please remove this line.

**Comment by floweisshardt** [comment]
CR: OK

File: cob_bringup/robots/cob4-1.xml [hunk 5]
```diff
@@ -231,6 +231,14 @@
 			<arg name="robot" value="$(arg robot)" />
 		</include>
 
+		<include file="$(find cob_bringup)/drivers/usb_camera_node.launch" />
+
+		<include file="$(find cob_bringup)/drivers/image_flip.launch">
+			<arg name="robot" value="$(arg robot)"/>
+			<arg name="pkg_hardware_config" value="$(arg pkg_hardware_config)"/>
```
Comment (floweisshardt) [inline cob_bringup/robots/cob4-1.xml]: @ipa-bnm: sorry, travis spottet another line which can be deleted

**Pull request merged**